modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -97,11 +97,11 @@ spec:
 +
 [source,terminal]
 ----
-$ oc get -o jsonpath=‘{.status.infrastructureName}{“\n”}’ infrastructure cluster
```

Comment:
The reason the previous command fails is that the quotation are full-width, but they should be half-width. 
the command `oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster` should work too
becasue the changed one by pr is also good, so the pr looks good to me.